README.md

```diff
@@ -1,72 +1,115 @@
 # stellar
 
-The `stellar` command line interface (CLI) is an __experimental__ tool primarily used by the StellarStation development team
-to perform internal end-to-end tests when developing the [StellarStation API](https://github.com/infostellarinc/stellarstation-api).
+The `stellar` command line interface (CLI) is an __experimental__ tool. Support is provided on a "best effort" basis. It is strongly recommended that users develop their own client-side software to interface with the StellarStation API.
 
-For best results, it is strongly recommended that users develop their own client interfaces which communicate directly
-with the defined StellarStation API, rather than through the `stellar` CLI tool.
+See the [documentation](/docs/stellar.md) for more details about commands.
```

Comment:
In the CLI, you can use:

```sh
stellar --help
```
to see documentation and commands that are available in your current version.